src/external/atoms_calculators.jl

```diff
@@ -69,12 +71,20 @@ AtomsCalculators.@generate_interface function AtomsCalculators.potential_energy(
         system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
         kwargs...)
     compute_scf!(system, calculator, state)
-    calculator.state.scfres.energies.total
+    calculator.state.scfres.energies.total * u"hartree"
 end
 
 AtomsCalculators.@generate_interface function AtomsCalculators.forces(
         system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
         kwargs...)
     compute_scf!(system, calculator, state)
-    compute_forces_cart(calculator.state.scfres)
+    compute_forces_cart(calculator.state.scfres) * u"hartree/bohr"
+end
+
+AtomsCalculators.@generate_interface function AtomsCalculators.virial(
+        system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
+        kwargs...)
+    compute_scf!(system, calculator, state)
+    - (compute_stresses_cart(calculator.state.scfres)
+     * calculator. state.scfres.basis.model.unit_cell_volume) * u"hartree"
```

Comment:
There is a space too much after the `calculator.`

Plus are you sure this is the right unit ? I had in mind it was something of the dimensions energy / volume, but I could be wrong.

Reply:
According to [AtomsCalculators](https://juliamolsim.github.io/AtomsCalculators.jl/dev/interface-definition/) interface definition: 
`Virial is a square matrix (3x3 in 3D) that has units of force times length or energy`, so here hartrees are appropriate.